objc/src/main/java/org/robovm/objc/ObjCObject.java

```diff
@@ -455,25 +507,25 @@ static void release(@Pointer long self, @Pointer long sel) {
             }
             ObjCRuntime.void_objc_msgSendSuper(new Super(self, NS_OBJECT_CLASS).getHandle(), sel);
```

Comment:
This should use the actual super class and not `NS_OBJECT_CLASS`. Use `ObjCRuntime.class_getSuperclass(...)`.